src/main/java/com/google/codeu/data/User.java

```diff
@@ -1,12 +1,27 @@
 package com.google.codeu.data;
 
+import java.util.ArrayList;
+import java.util.UUID;
+
 public class User {
 
   private String email, aboutMe;
+  private ArrayList<UUID> mentions;
 
   public User(String email, String aboutMe) {
     this.email = email;
     this.aboutMe = aboutMe;
+    this.mentions = new ArrayList<UUID>();
+  }
+
+  public User(String email, String aboutMe, List mentions) {
+    this.email = email;
+    this.aboutMe = aboutMe;
+    if (this.mentions != null) 
```

Comment:
This may not be necessary and I'll delete if that becomes the case; just made this so I wouldn't have to worry about it later.